INSTALLATION.md

```diff
@@ -1,13 +1,72 @@
-```
-cd dataviz/provisioning
+# EEA Grants Dataviz installation
 
-cp salt-config/minion.example salt-config/minion
-cp salt-pillar/settings.sls.example salt-pillar/settings.sls
-# edit them if necessary ^^
+These instructions assume you're deploying to Azure Containers, and have already set up an account on the Azure portal.
 
-salt-call state.apply
+1. Install Azure CLI: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli
 
-# if you edit dv/localsettings.py don't forget to 
-systemctl restart uwsgi-dataviz.service
+1. Set up an Azure context in Docker:
+    ```shell
+    docker login azure
+    docker context create aci eeagstaging
+    source docker/set_context.sh eeagstaging
+    ```
 
-```
+1. Configure environment variables – copy and modify the examples:
+    ```shell
+    cp docker/web.prod.env.example docker/web.env
+    cp docker/azure-nginx.env.example docker/azure-nginx.env
+    ```
+
+1. Create volumes:
+    ```shell
+    docker volume create --storage-account eeagstorage solrhome
```

Comment:
Don't you need to create the storage account first?

Reply:
No, it's created automatically.